media/admin/com_virtuemart/models/zasilkovna.php

```diff
@@ -9,12 +9,23 @@
 
 if(!class_exists('VmModel')) require(VMPATH_ADMIN . DS . 'helpers' . DS . 'vmmodel.php');
 
+spl_autoload_register(
```

Comment:
Nepatřilo by to do samostatného souboru i s obsahem konstruktoru plgVmShipmentZasilkovnaInstallerScript ?

Reply:
Mělo by to být v autoload.php, ale nevěděl jsem kam ho dát. Nápady? Obsah konstruktoru plgVmShipmentZasilkovnaInstallerScript s tím nesouvisí. Na ten obsah bych nešahal. Použít ho i tady jsem zkoušel.

Reply:
v rootu v zasilkovna.php je duplicitní kód, umístil bych to do rootu